nat.rst

```diff
@@ -47,6 +47,14 @@ select SNAT as action.
 **Result** All outbound traffic originating from your mail server will now be translated to the dedicated public IP address.
 This improves the reputation of your mail server and allows for specific configurations tailored to its needs. General internet traffic will continue to use the other public IP address.
 
+Source NAT in a MultiWAN scenario
+---------------------------------
+
+If you have multiple WANs, you need to create a MultiWAN rule in addition to the SNAT rule. This rule will route traffic from the source IP address through the WAN with the public IP address.
```

Comment:
```suggestion
If you have multiple WANs and your SNAT rule rewrites to one of the public WAN IPs, you need to create a MultiWAN rule in addition to the SNAT rule. This rule will route traffic from the source IP address through the right WAN with the public IP address.
```

Reply:
I think it's redundant if you read the manual from the beginning of the chapter.

Reply:
It may be, but we are not sure that the chapter has been read from the beginning, so I'd prefer to specify it and be sure that the information is well understood.